var/spack/repos/builtin/packages/gpi-2/package.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+import os
+from spack import *
+
+
+def _verbs_dir():
```

Comment:
I think we used to have something like this in the OpenMPI package, but we removed it for several reasons.

Reply:
You are right, this one was based on OpenMPI. I have removed it as suggested!